scripts/update_agent_versions.py

```
@@ -0,0 +1,284 @@
+)
+
+
+def validate_version(version: str, package: str) -> None:
+    if not STABLE_VERSION_RE.fullmatch(version):
+        raise ValueError(
+            f"{package} latest must be a stable semantic version, got {version!r}"
+        )
+
+
+def fetch_latest_version(package: str) -> str:
+    completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)